src/Questionnaire.js

```diff
@@ -1,118 +1,124 @@
-      }); 
+      const matchRequest = new Request("http://127.0.0.1:5000/potentialMatch", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ responses: this.state.response, userId: id }),
+      });
 
       fetch(myRequest)
-      .then((res) =>
-        res.json())
-      .then((res) =>{
-        if(res.response === "Success") {
-          //Once the first questionnaire API call is a success, the second matching call is nested to ensure that the two calls
-          //happen in succession
-          fetch(matchRequest)
-          .then((res) =>
-            res.json())
-          .then((res) => {
-            if(res.response === "Success"){
-              this.props
-              .history.push({
-                pathname: "/main",
-                state: {id: id}
+        .then((res) => res.json())
+        .then((res) => {
+          if (res.response === "Success") {
+            //Once the first questionnaire API call is a success, the second matching call is nested to ensure that the two calls
+            //happen in succession
+            fetch(matchRequest)
+              .then((res) => res.json())
+              .then((res) => {
+                if (res.response === "Success") {
+                  this.props.history.push({
+                    pathname: "/main",
+                    state: { id: id },
+                  });
+                } else {
+                  this.setState({
+                    error: res.response,
+                  });
+                }
               });
+          } else {
+            this.setState({
+              error: res.response,
+            });
           }
-            else {
-              this.setState({
-                error: res.response,
-              });
-            }
-          })
-        }
-        else {
+        })
+
+        .catch((error) => {
           this.setState({
-            error: res.response,
+            error: "Error connecting to backend",
           });
-      }
-      })
-      
-      .catch((error) =>{
-        this.setState({
-          error: "Error connecting to backend",
         });
-      });
-    }
-    else{
+    } else {
       alert("Please answer all questions");
     }
-  };
+  }
 
   getQuestions = () => {
-    questions().then(question => {
+    questions().then((question) => {
       this.setState({
-        questionBank: question
+        questionBank: question,
       });
     });
   };
   componentDidMount() {
     this.getQuestions();
   }
-  storeAnswer = (answer,ID) => {
-    this.state.response[ID-1]=answer;
-  }
+  storeAnswer = (answer, ID) => {
+    this.state.response[ID - 1] = answer;
+  };
 
-  render(){
-    return(
-      <div>
-        <div>
+  render() {
+    return (
+      <div className="questionnaireBox">
+        <div className="header">
+          <br></br>
+          <br></br>
+          <img src="question-penguin.svg"></img>
```

Comment:
Please refer to the wireframe that was created and change design accordingly. 

![Survey](https://user-images.githubusercontent.com/48136962/99679483-fec16b00-2a49-11eb-98c2-c6ef77380b3b.png)

Reply:
you make the scroll through a set height and overflow: auto

Reply:
added scrolling feature, but left the radio buttons to the bottom of each question

Reply:
good job joy!